cmd/fortifyExecuteScan.go

```diff
@@ -508,11 +509,9 @@ func triggerFortifyScan(config fortifyExecuteScanOptions, command execRunner, bu
 		if config.AutodetectClasspath {
 			classpath = autoresolveMavenClasspath(config.BuildDescriptorFile, classpathFileName, command)
 		}
-		if len(config.Translate) == 0 {
-			translate := `[{"classpath":"`
-			translate += classpath
-			translate += `","src":"**/*.xml **/*.html **/*.jsp **/*.js src/main/resources/**/* src/main/java/**/*"}]`
-			config.Translate = translate
+		config.Translate, err = populateMavenTranslate(&config, classpath)
+		if err != nil {
+			log.Entry().WithError(err).Warnf("failed to read src or exclude from project configuration source: '%s', exclude: '%s'", config.Src, config.Exclude)
```

Comment:
```suggestion
			log.Entry().WithError(err).Warnf("failed to apply src ('%v') or exclude ('%v') parameter", config.Src, config.Exclude)
```

Suggestion mainly motivated because of naming the 'src' parameter 'source: ' in the output, but then I realized that the parameter may not only come from the "project configuration", but also via command line or defaults.